sunpy/timeseries/timeseries_factory.py

```diff
@@ -238,8 +236,7 @@ def _sanitise_args(self, args):
         Sanitise a list of args so that a single argument corresponds to either:
 
         - A (data, header, units) tuple
-        - A path-like string (e.g. a filename, directory, glob etc.)
-        - A URL
```

Comment:
Is this still not true?

Reply:
I updated this to mirror what `MapFactory` does, ie. convert URLs to a request and then use that to dispatch on.